README.md

```diff
@@ -43,14 +64,20 @@ The reducer can be functions such as `mean`, `median`, `length` but can also tak
 ```julia
 #calculate raster of median height using an anonymous function
 height_percentile = reduce(raster_index, field=:Z, reducer = x -> quantile(x,0.5))
+```
+
+`field` is always a symbol and can either be `:X`, `:Y`, or `:Z`. In the event that your area of interest and/or cellsize is square, using `:X` or `:Y` may both return the same results.
 
-#calculate raster of point density per cell
-counts_x = reduce(raster_index, field=:X, reducer=length)
+Any reduced layer is returned as a [GeoArray](https://github.com/evetion/GeoArrays.jl) and has the same fields. 
```

Comment:
What do you mean by "has the same fields"? I'd say something like, the values in the GeoArray cells are equal to that returned by the reducer function, which operates on the given field of all points that lie in that cell. Or something like that :)

Reply:
I meant it as the result type is a `GeoArray`, so the underlying structs are detailed in that package. I wrote this largely assuming someone found this without knowing about other packages. Not sure how much to assume, maybe just the link to the other README is enough?

`julia> typeof(counts_x)
GeoArray{Union{Missing, Float64}}`

`julia> fieldnames(typeof(counts_x))
(:A, :f, :crs)`

Reply:
I think it's good not to assume people know GeoArrays, so I think the text below is a good addition. I just thought you were talking about the same fields as the pointcloud, not the GeoArray.